server/youtube_videos.py

```diff
@@ -24,6 +24,35 @@ def youtube_search(q, max_results=50,order="relevance", token=None, location=Non
 
```

Comment:
Please convert this file in a class and follow OOP principles in the other files as well